pkg/render/intrusion_detection.go

```diff
@@ -106,7 +118,12 @@ func (c *intrusionDetectionComponent) ResolveImages(is *operatorv1.ImageSet) err
 		errMsgs = append(errMsgs, err.Error())
 	}
 
-	c.adJobsImage, err = components.GetReference(components.ComponentAnomalyDetectionJobs, reg, path, prefix, is)
+	c.adDetectorsImage, err = components.GetReference(components.ComponentAnomalyDetectionJobs, reg, path, prefix, is)
```

Comment:
Both this and the adAPIImage are only needed when `!c.cfg.ManagedCluster` like the `jobInstallerImage` as far as I can tell. Is that correct?